ui/media/css/main.css

```diff
@@ -400,6 +409,17 @@ img {
 .secondaryButton:hover {
     background: rgb(177, 27, 0);
 }
+.useSettings {
+	background: var(--accent-color);
```

Comment:
Please use spaces instead of tabs :)

Reply:
Yeah, I noticed that and changed the tab behavior in my text editor since then. I cleaned up all my tab tab leftovers and then some, especially in this file. :)